packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -528,9 +573,22 @@ async function createRedwoodApp() {
   const useGit = await handleGitPreference(gitInitFlag)
   trace.getActiveSpan()?.setAttribute('git', useGit)
 
+  /** @type {string} */
+  let commitMessage
+  if (useGit) {
+    commitMessage = await handleCommitMessagePreference(commitMessageFlag)
+  }
```

Comment:
I know this if (useGit) could be combined with the one below, but right now at least I prefer to have this script separated into a "gathering preferences" section and a "executing on those preferences" section